### PR TITLE
Powerplantmatching

### PR DESCRIPTION
**Pull request by FabianHofmann**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/powerplantmatching/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - networkx>=1.10
+    - pycountry
+    - xlrd
+    - seaborn
+    - pyyaml
+    - requests
+    - matplotlib
+    - geopy
+    - pip:
+      - entsoe-py
```
Comment (scopatz) [inline recipes/powerplantmatching/meta.yaml]: This should be packaged on conda-forge first

File: recipes/powerplantmatching/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - numpy
+    - scipy
+    - pandas>=0.23.0
+    - networkx>=1.10
```
Comment (scopatz) [inline recipes/powerplantmatching/meta.yaml]: These need a single space, e.g. `pandas >=0.23.0`

File: recipes/powerplantmatching/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    a new combining dataset, which includes all the important information. 
+    The package allows to easily update the combined data as 
+    soon as new input datasets are released.
+  doc_url: https://github.com/FRESNA/powerplantmatching
+  dev_url: https://github.com/FRESNA/powerplantmatching
+
+extra:
+  recipe-maintainers:
+    - FabianHofmann
+    - coroa
```
Comment (scopatz) [inline recipes/powerplantmatching/meta.yaml]: @coroa, are you OK with being a maintainer?

File: recipes/powerplantmatching/meta.yaml (hunk 1 above)
Comment (FabianHofmann) [reply recipes/powerplantmatching/meta.yaml]: Thank you for reviewing, then I will set up a conda-forge package for entsoe-py first

File: recipes/powerplantmatching/meta.yaml (hunk 3 above)
Comment (coroa) [reply recipes/powerplantmatching/meta.yaml]: Yes, thanks!

**Comment by FabianHofmann** [comment]
dear @scopatz, can I kindly ask you to merge this? I think all problems are resolved

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**